README.md

```diff
@@ -1354,7 +1354,7 @@ $ yum install trivy
 ```
 
 ## Debian/Ubuntu
-Use https://aquasecurity.github.io instead of https://knqyf263.github.io.
+Use https://aquasecurity.github.io/trivy-repo/ instead of https://knqyf263.github.io.
```

Comment:
This is a migration section. A user only needs to change the domain. This change is unnecessary.

Reply:
So could you tell me what i should do? Just replace this part with the previous?

Reply:
Yes. You don't have to replace it.